lib/Extractor/Candidates.cpp

```diff
@@ -563,6 +563,11 @@ Inst *ExprBuilder::buildHelper(Value *V) {
           Inst *Overflow = IC.getInst(Inst::UMulO, 1, {L, R}, /*Available=*/false);
           return IC.getInst(Inst::UMulWithOverflow, L->Width+1, {Mul, Overflow});
         }
+        case Intrinsic::sadd_sat: {
+	  // TODO: Factor out *R
+	  Inst *R = get(II->getOperand(1));
+	  return IC.getInst(Inst::SAddSat, L->Width, {L, R}, /*Available=*/false);
```

Comment:
why false for available?

Reply:
i am not sure really yet how it affects things. Yeah, from the pattern of other instructions above, it should be true (which is also the default if i don't mention it).

Reply:
should be true

Reply:
this is outdated. i have already fixed that.